README.md

```diff
@@ -24,6 +24,13 @@ All messages                         | Alpha    |
 (See [maturity-matrix.yaml](https://github.com/open-telemetry/community/blob/47813530864b9fe5a5146f466a58bd2bb94edc72/maturity-matrix.yaml#L57)
 for definition of maturity levels).
 
+Note that maturity guarantees apply only to wire-level compatibility. Neither message,
+field, nor enum names of Protobuf messages are visible on the wire and are not
+considered part of the guarantees. We are free to make change to the names.
+
+In the future when OTLP/JSON is declared stable, field names will also become part of
```

Comment:
I think OTLP/JSON should just always be left unstable. I think the cost/benefit ratio to declaring this stable is just not right. Are there any strong reasons for supporting JSON at all?

Reply:
Short answer: yes, we want it, but I am not sure we want to open this discussion here in this PR. OTLP/JSON is already part of the specification: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/otlp.md#request

Reply:
I think @Oberon00 has good arguments (probably more in his mind). Would be good to have this discussion, also may be good to discuss if JSON should be based on these protos or we defined it separately.